src/main/java/net/syscon/elite/repository/jpa/model/Address.java

```diff
@@ -43,8 +45,13 @@
 
     private String flat;
 
-    @Column(name = "ADDRESS_TYPE")
-    private String addressType;
+    @ManyToOne
```

Comment:
Should this be FetchType.LAZY?

Reply:
Oooh, I thought it was lazy by default? We don't seem to specify it in most places.

Reply:
Apparently only for @OneToMany - mentioned in this thread on Slack https://mojdt.slack.com/archives/C012FNNQ4A3/p1591606149012500

Reply:
that comment mentions about always resolving the data, I think that's appropriate for reference codes as we'd always want to have the description

Reply:
I think there's only a problem where some entity data does is not included on the DTO - queries running for no reason.

Reply:
Generally we don't want EAGER because JPA will always join to these tables and we might not be interested in them.  It's better to let the application load what it needs.  The caveat on that is that a big list might want to join to one extra table (say its parent) for example if we were selecting a 2000 bookings we might want to inner join to offenders table because we always need their name.  However JPA does allow you to explicitly join in its query language on specific repo calls so better to leave the default to LAZY